_auto/validate.py

```diff
@@ -41,7 +41,6 @@ def validate_product(name):
             assert isinstance(data["auto"], comments.CommentedSeq)
         assert data["category"] in VALID_CATEGORIES
         assert data["permalink"][0] == "/"
-        assert isinstance(data["iconSlug"], str)
```

Comment:
For a future PR: We could easily add a test that validates these icons against the simple-icons DB. Or perhaps they can be caught by a broken-link detector as well.